src/midi/fluid_midi.c

```diff
@@ -1202,6 +1202,13 @@ fluid_track_send_events(fluid_track_t *track,
 {
     int status = FLUID_OK;
     fluid_midi_event_t *event;
+    int seeking = player->seek_ticks >= 0;
+
+    if (seeking) {
+        ticks = player->seek_ticks; /* update target ticks */
+        if (track->ticks > ticks)
+            fluid_track_reset (track); /* reset track if seeking backwards */
```

Comment:
Seems to work. But you should call `fluid_synth_all_sounds_off()` in this if clause to avoid hanging notes.